cmake/simdjson-flags.cmake

```diff
@@ -86,7 +86,9 @@ else()
 endif()
 if(SIMDJSON_VISUAL_STUDIO_BUILD_WITH_DEBUG_INFO_FOR_PROFILING)
   target_link_options(simdjson-flags  INTERFACE    /DEBUG ) 
-  target_compile_options(simdjson-flags INTERFACE  /Zi    )
+  target_compile_options(simdjson-flags INTERFACE  /Zi)
+  target_link_options(simdjson-internal-flags  INTERFACE    /DEBUG ) 
```

Comment:
The simdjson-internal-flags lines *should* be unnecessary, since simdjson-internal-flags inherits from simdjson-flags. Did you find otherwise?

Reply:
You are correct. They were unnecessary. Removed.